lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1466,7 +1466,7 @@ class JsonInitDataTest(ModuleStoreTestCase):
 
     @ddt.data(
         ({'a': 17}, '''{"a": 17}'''),
-        ({'xss': '</script>alert("XSS")'}, r'''{"xss": "<\/script>alert(\"XSS\")"}'''),
+        ({'xss': '</script>alert("XSS")'}, r'''{"xss": "\u003c/script\u003ealert(\"XSS\")"}'''),
```

Comment:
Sure. 
We pass json data from html to js code for different use cases, in this case as parameters to initialize a js object. We want this json to be taken as key/value pair of string literals instead of html. To do so we are escaping any html tags present in json.
Please let me know if you need further details.